test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -214,12 +214,12 @@ test
 
     const pageTitle = challengeEmailPageObject.getFormTitle();
     const saveBtnText = challengeEmailPageObject.getSaveButtonLabel();
-    await t.expect(pageTitle).eql('Verify with your email');
+    await t.expect(pageTitle).eql('Get a verification email');
     await t.expect(saveBtnText).eql('Send me an email');
 
     const emailAddress = emailVerificationSendEmailData.currentAuthenticatorEnrollment.value.profile.email;
     await t.expect(challengeEmailPageObject.getFormSubtitle())
-      .eql(`Verify with an email link or enter a code sent to ${emailAddress}.`);
+      .eql(`Send a verification email to ${emailAddress} by clicking on "Send me an email".`);
```

Comment:
nit: can we create constants for the strings since you check for it multiple times in the file

Reply:
Line 217 can be extracted out but not this one as this is only once.

Reply:
Done